src/applications/claims-status/containers/DocumentRequestPage.jsx

```diff
@@ -84,9 +77,7 @@ class DocumentRequestPage extends React.Component {
       document.querySelector('.claims-alert').focus();
       scrollToError();
     }
-    if (!this.props.loading && prevProps.loading) {
-      setPageFocus();
-    }
```

Comment:
I think we should keep this?  Maybe put the `setPageTitle(this.props.trackedItem)` inside of this if statement?